app/AndroidManifest.xml

```diff
@@ -67,9 +67,8 @@
                 <action android:name="android.intent.action.VIEW"/>
                 <category android:name="android.intent.category.DEFAULT"/>
                 <category android:name="android.intent.category.BROWSABLE"/>
-                <data android:scheme="https"
-                  android:host="appauth.demo-app.io"
-                  android:path="/oauth2redirect"/>
+                <data android:scheme="appscheme"
```

Comment:
The comment I added above this declaration explains that this is a demonstration of how to capture additional redirect URIs, and in particular I'm trying to show how to capture an app link (registered https redirect). To capture a custom scheme, the developer can just change the `appAuthRedirectScheme` manifest placeholder.